client/ayon_core/plugins/publish/collect_hierarchy.py

```diff
@@ -32,36 +31,49 @@ def process(self, context):
             product_type = instance.data["productType"]
             families = instance.data["families"]
 
-            # exclude other families then self.families with intersection
-            if not set(self.families).intersection(
-                set(families + [product_type])
-            ):
+            # exclude other families then "shot" with intersection
+            if "shot" not in (families + [product_type]):
+                self.log.debug("Skipping not a shot: {}".format(families))
                 continue
 
-            # exclude if not masterLayer True
+            # Skip if is not a hero track
             if not instance.data.get("heroTrack"):
+                self.log.debug("Skipping not a shot from hero track")
                 continue
 
             shot_data = {
                 "entity_type": "folder",
-                # WARNING Default folder type is hardcoded
-                # suppose that all instances are Shots
-                "folder_type": "Shot",
+                # WARNING unless overwritten, default folder type is hardcoded to shot
```

Comment:
yeah lets keep it as it is. Thanks https://github.com/ynput/ayon-traypublisher/pull/36#issuecomment-2488175999